kod/util/assgame.kod

```diff
@@ -70,6 +70,11 @@ resources:
    AssGame_mail_winner = "Subject: One has risen above the others.\n"
       "%q has won the challenge of the dark blades.\n\n"
       "-- Roq, the Dark Blade"
+
+   AssGame_not_viable = "Subject: insufficient combatants.\n"
+      "The Assassin's Game is not viable at this time.  The game will "
```

Comment:
"not viable at this time" is pretty vague.  Let's actually tell the player why the game didn't start.